UnrealTournament/SystemLocalized/itt/Botpack.itt

```diff
@@ -3107,11 +3108,14 @@ ItemName="Caricatore"
 ItemName="Enforcer Doppi"
 
 [Translocator]
-; EN: WeaponDescription="Classification: Personal Teleportation Device\n\nPrimary Fire: Launches the destination module.  Throw the module to the location you would like to teleport to.\n\nSecondary Fire: Activates the translocator and teleports the user to the destination module.\n\nTechniques: Throw your destination module at another player and then activate the secondary fire, and you will telefrag your opponent!  If you press your primary fire button when activating your translocator with the secondary fire, the last weapon you had selected will automatically return once you have translocated."
```

Comment:
I'm curious, why the change of line here?

Reply:
This was a really strange bug: in the weapon description window, the translocator had missing title (the english one was shown) and the description was cutted (was in italian but incomplete). Trying to understand why, I discovered that ordering those lines in diverse order in Botpack.itt file, the title was correctly translated, but the description was cutted ever more.

Trying, I put more padding using those comments, and then the Translocator window was perfect: italian title and the whole description shown. This is why I put those padding lines.

Reply:
That's indeed weird. Let me check it out.